airbyte-cdk/java/airbyte-cdk/build.gradle

```diff
@@ -26,6 +26,9 @@ allprojects {
         testCompileOnly libs.lombok
         testAnnotationProcessor libs.lombok
 
+        // Pin to 1.13.6 due to https://github.com/mockk/mockk/issues/1133
```

Comment:
looks like this was fixed in master 1h ago!

Reply:
Awesome.  I'll test the fix out (which looks exactly like the fix I was going to attempt to test locally) once they publish a new version and update/remove the comment if it works.  We should not hold up merging this if it takes them a few days to cut a release, as what we currently have works.